data-access-layer/iaas/CloudProviderClient.js

```diff
@@ -61,10 +61,10 @@ class CloudProviderClient extends BaseCloudClient {
   list(container, options) {
     if (arguments.length < 2) {
       options = container;
-      container = this.containerName;
+      container = _.get(options, 'container') || this.containerName;
```

Comment:
May be not required this. call **list(container, options)** method with container [here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/355/files#diff-c79d1f6f368d710e95c0626775567787R50)

Reply:
Yes, container is already passed as param, why do we need it in options again?

Reply:
Not sure if I understand this, you can pass container as parameter instead if passing in options.

Reply:
Agreed. I think we can get rid of this logic and retrieve the `container` from the function param.
I have made the changes accordingly.